Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<ul><li>Maybe you can add a legend to indicate the meaning of blue and unfilled points?</li><li>In the version of the book I have, the vertical axis is <code>happiness</code> , so it goes from -2 to 2</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>